modules/cluster-logging-update-logging.adoc

```diff
@@ -0,0 +1,7 @@
+[id="cluster-logging-update-logging-about_{context}"]
+= About updating logging
+
+{product-title} allows you to update {product-title} logging. You must update the following operators as a part of updating {product-title} logging:
```

Comment:
s/logging/{product-title} Logging

As part of? I'm left wondering what else I need to do ...

Reply:
Removed `as a part of`